cmd/nginx/flags.go

```diff
@@ -179,6 +182,7 @@ Takes the form "<host>:port". If not provided, no admission controller is starte
 	flags.StringVar(&nginx.MaxmindLicenseKey, "maxmind-license-key", "", `Maxmind license key to download GeoLite2 Databases.
 https://blog.maxmind.com/2019/12/18/significant-changes-to-accessing-and-using-geolite2-databases`)
 	flags.StringVar(&nginx.MaxmindEditionIDs, "maxmind-edition-ids", "GeoLite2-City,GeoLite2-ASN", `Maxmind edition ids to download GeoLite2 Databases.`)
+	flags.IntVar(&nginx.MaxmindRetriesTimeout, "maxmind-retries-timeout", 0, "Maxmind download retries timeout in seconds, 0 - do not retry to download if something went wrong")
```

Comment:
If this is a duration it should be `flags.DurationVar()`.

Reply:
Fixed